sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -150,23 +162,26 @@ def update_room(
         self,
         *,
         room_id: str,
-        valid_from: datetime,
-        valid_until: datetime,
+        valid_from: Optional[datetime] = None,
+        valid_until: Optional[datetime]= None,
+        pstnDialOutEnabled: Optional[bool] = None,
         **kwargs: Any
     ) -> CommunicationRoom:
         """Update a valid room's attributes. For any argument that is passed
         in, the corresponding room property will be replaced with the new value.
 
         :keyword str room_id: Required. Id of room to be updated
-        :keyword datetime valid_from: Required. The timestamp from when the room is open for joining.
-        :keyword datetime valid_until: Required. The timestamp from when the room can no longer be joined.
+        :keyword datetime valid_from: The timestamp from when the room is open for joining. Optional.
+        :keyword datetime valid_until: The timestamp from when the room can no longer be joined. Optional.
+        :keyword bool pstnDialOutEnabled: Set this flag to true if, at the time of the call, dial out to a PSTN number is enabled in a particular room. Optional.
         :returns: Updated room.
         :rtype: ~azure.communication.rooms.CommunicationRoom
         :raises: ~azure.core.exceptions.HttpResponseError, ValueError
         """
         update_room_request = {
             "validFrom": valid_from,
-            "validUntil": valid_until
+            "validUntil": valid_until,
+            "pstnDialOutEnabled": pstnDialOutEnabled,
```

Comment:
If we send a null value over the wire here, is that the same as not sending/updating the value at all? (i.e. the service will interpret null to mean that the attribute will not be updated).

Reply:
correct